web-console/src/lib/components/streaming/management/PipelineTable.tsx

```diff
@@ -565,7 +536,12 @@ export default function PipelineTable() {
     </Button>
   )
 
-  const fetchRows = useQuery({ initialData: rows })
+  const [expandedRows, setExpandedRows] = useState<GridRowId[]>([])
```

Comment:
might be better to rely on `useLocalStorage` and remember across sessions which one is expanded